src/winter_onboarding_2021/fleet_management_service/models/cab.clj

```diff
@@ -0,0 +1,21 @@
+(ns winter-onboarding-2021.fleet-management-service.models.cab
+  (:require  [clojure.spec.alpha :as s]
+             [winter-onboarding-2021.fleet-management-service.db.core :as db]
+             [winter-onboarding-2021.fleet-management-service.models.core :as m-core]))
+
+(s/def :cab/name string?)
```

Comment:
Umm, not right now. Govind [suggested](https://github.com/nilenso/winter-onboarding-2021/pull/21#discussion_r758041280) that I should move the cab specs to cab model file so that anyone new can know about cab in one place.